src/components/inputs/toggle/Toggle.tsx

```diff
@@ -0,0 +1,74 @@
+import { motion } from 'framer-motion';
+import { useState } from 'react';
+
+function Toggle({
+  leftLabel,
+  rightLabel,
+  check,
+  className,
+  inputClassName,
+}: {
+  leftLabel: string;
+  rightLabel: string;
+  check: (b: boolean) => void;
+  className?: string;
+  inputClassName?: string;
+}) {
+  const [checked, setChecked] = useState(false);
```

Comment:
Rather than have a local state and a setter coming in, think it'd be better for this component to take in checked and setChecked and remove useState here. It'd make it more inline with other input components.

Reply:
Decided to remove this component completely for now per design recs